cmd/root.go

```diff
@@ -141,4 +143,5 @@ func init() {
 	// TODO(MK-1408): This value is highly dependend on underlying hardware, thus making the default value a bit useless. The linked card will implement a dynamic way to set this value.
 	rootCmd.Flags().Int64Var(&rootCmdOpts.acpLimitMaxConcurrentTxn, "admission-control-policy-limit-max-concurrent-transactions", 300, "Maximum number of transactions that are allowed to run concurrently. Transactions will not be admitted after the limit is reached.")
 	rootCmd.Flags().BoolVar(&rootCmdOpts.acpOnlyWriteQueries, "admission-control-only-for-write-queries", false, "If set, admission control will only be applied to write queries.")
+	rootCmd.Flags().DurationVar(&rootCmdOpts.pollAfterTimeout, "poll-after-timeout", 20*time.Second, "Timeout on the after query during the poll loop. If timeout is reached, the poll loop will be re-triggered")
```

Comment:
```suggestion
	rootCmd.Flags().DurationVar(&rootCmdOpts.pollAfterTimeout, "poll-events-timeout", 20*time.Second, "Timeout for querying events in the watch poll loop. If timeout is reached, the loop will be re-triggered")
```

Can we rename the parameters to be more descriptive? The `After` keyword does not mean anything to normal users.

Reply:
Great point, thanks Berkay!